src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -233,7 +258,7 @@ const JobHistory = _.flow(
               cellRenderer: ({ rowIndex }) => {
                 const { workflowStatuses, status } = filteredSubmissions[rowIndex]
                 return h(Fragment, [
-                  statusCell(workflowStatuses), _.keys(collapsedStatuses(workflowStatuses)).length === 1 && status
+                  statusCell(workflowStatuses, status)
```

Comment:
Ah. I moved the conditional logic into the `statusCell` function since I needed the content to be nested rather than parallel. I'll remove the `Fragment` since there's now only one child here